indra/llprimitive/llmaterial.h

```diff
@@ -115,8 +115,6 @@ class LLMaterial : public LLRefCount
     void        setDiffuseAlphaMode(U8 alpha_mode);
     U8          getAlphaMaskCutoff() const;
     void        setAlphaMaskCutoff(U8 cutoff);
-    LLUUID      getMaterialID() const;
-    void        setMaterialID(LLUUID const & material_id);
```

Comment:
We will also want to ensure these changes get into server side llprimitive project as well to keep them from diverging too much further